databricks/koalas/indexes.py

```diff
@@ -571,6 +571,35 @@ def __getattr__(self, item: str) -> Any:
     def rename(self, name, inplace=False):
         raise NotImplementedError()
 
+    @property
+    def levels(self) -> list:
+        """
+        Names of index columns in list.
+
+        .. note:: Be aware of the possibility of running into out
+        of memory issue if returned list is huge.
+
+        Examples:
+        --------
```

Comment:
nit: need one more `-`?

Reply:
I removed `:` instead to align with other docstring. And add it in `doc/source`